internal/olm/client.go

```diff
@@ -104,7 +107,7 @@ func (c Client) InstallVersion(ctx context.Context, version string) (*olmresourc
 	})
 
 	for _, sub := range subscriptions {
-		subscriptionKey := types.NamespacedName{Namespace: sub.GetNamespace(), Name: sub.GetName()}
+		subscriptionKey := types.NamespacedName{Namespace: namespace, Name: sub.GetName()}
```

Comment:
Before it was getting the namespace from the subscriptions. I do not think that it should be changed.

Reply:
I agree. I'l change it back.

Infact I find no subscriptions resources in the olm install. If the notice the log, its never called.

```
$ operator-sdk olm install --version 0.14.1
INFO[0000] Fetching CRDs for version "0.14.1"
INFO[0001] Fetching resources for version "0.14.1"
INFO[0007] Creating CRDs and resources
INFO[0007]   Creating CustomResourceDefinition "clusterserviceversions.operators.coreos.com"
INFO[0007]   Creating CustomResourceDefinition "installplans.operators.coreos.com"
INFO[0007]   Creating CustomResourceDefinition "subscriptions.operators.coreos.com"
INFO[0007]   Creating CustomResourceDefinition "catalogsources.operators.coreos.com"
INFO[0007]   Creating CustomResourceDefinition "operatorgroups.operators.coreos.com"
INFO[0007]   Creating Namespace "olm"
INFO[0007]   Creating Namespace "operators"
INFO[0007]   Creating ServiceAccount "olm/olm-operator-serviceaccount"
INFO[0007]   Creating ClusterRole "system:controller:operator-lifecycle-manager"
INFO[0007]   Creating ClusterRoleBinding "olm-operator-binding-olm"
INFO[0008]   Creating Deployment "olm/olm-operator"
INFO[0008]   Creating Deployment "olm/catalog-operator"
INFO[0008]   Creating ClusterRole "aggregate-olm-edit"
INFO[0008]   Creating ClusterRole "aggregate-olm-view"
INFO[0008]   Creating OperatorGroup "operators/global-operators"
INFO[0009]   Creating OperatorGroup "olm/olm-operators"
INFO[0009]   Creating ClusterServiceVersion "olm/packageserver"
INFO[0009]   Creating CatalogSource "olm/operatorhubio-catalog"
INFO[0009] Waiting for deployment/olm-operator rollout to complete
INFO[0010]   Waiting for Deployment "olm/olm-operator" to rollout: 0 of 1 updated replicas are available
INFO[0033]   Deployment "olm/olm-operator" successfully rolled out
INFO[0033] Waiting for deployment/catalog-operator rollout to complete
INFO[0033]   Deployment "olm/catalog-operator" successfully rolled out
INFO[0033] Waiting for deployment/packageserver rollout to complete
INFO[0033]   Waiting for Deployment "olm/packageserver" to rollout: 0 of 2 updated replicas are available
INFO[0041]   Deployment "olm/packageserver" successfully rolled out
INFO[0041] Successfully installed OLM version "0.14.1"

NAME                                            NAMESPACE    KIND                        STATUS
clusterserviceversions.operators.coreos.com                  CustomResourceDefinition    Installed
installplans.operators.coreos.com                            CustomResourceDefinition    Installed
subscriptions.operators.coreos.com                           CustomResourceDefinition    Installed
catalogsources.operators.coreos.com                          CustomResourceDefinition    Installed
operatorgroups.operators.coreos.com                          CustomResourceDefinition    Installed
olm                                                          Namespace                   Installed
operators                                                    Namespace                   Installed
olm-operator-serviceaccount                     olm          ServiceAccount              Installed
system:controller:operator-lifecycle-manager                 ClusterRole                 Installed
olm-operator-binding-olm                                     ClusterRoleBinding          Installed
olm-operator                                    olm          Deployment                  Installed
catalog-operator                                olm          Deployment                  Installed
aggregate-olm-edit                                           ClusterRole                 Installed
aggregate-olm-view                                           ClusterRole                 Installed
global-operators                                operators    OperatorGroup               Installed
olm-operators                                   olm          OperatorGroup               Installed
packageserver                                   olm          ClusterServiceVersion       Installed
operatorhubio-catalog                           olm          CatalogSource               Installed
```

Reply:
Solved,